dcp_prototype/backend/browser/api/.chalice/policy-test.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
```

Comment:
Is there an opportunity to scope this resource down further?

Reply:
:+1: will do. Will be addressed in @Bento007 's PR where chalice configs are generated